perllib/FixMyStreet/Roles/Cobrand/KingstonSutton.pm

```diff
@@ -97,9 +97,6 @@ sub available_permissions {
 
 sub waste_auto_confirm_report { 1 }
 
-sub waste_staff_choose_payment_method { 1 }
-sub waste_cheque_payments { shift->{c}->stash->{staff_payments_allowed} }
-
 use constant CONTAINER_REFUSE_140 => 1;
```

Comment:
I wonder if any of these can be removed from here / refactored into SLWP.pm, as there seems to be a fair bit of overlap.

Reply:
Yeah, is true, I wondered if I could put the constants somewhere, but I'm not sure how I'd easily refer to them (I wouldn't want to put $self-> in front of them all, for example). Probably technically should have some form of class to handle the containers or something, but didn't seem worth it.